modules/administration-guide/attachments/migration/3-subscribe.sh

```diff
@@ -52,19 +57,6 @@ patchPreMigrationCheCluster() {
       "${K8S_CLI}" patch checluster/"${PRE_MIGRATION_PRODUCT_CHE_CLUSTER_CR_NAME}" -n "${INSTALLATION_NAMESPACE}" --type=json -p \
               '[{"op": "replace", "path": "/spec/server/cheHost", "value": ""}]'
     fi
-
-    echo "[INFO] Updating ${PRE_MIGRATION_PRODUCT_CHE_CLUSTER_CR_NAME} CheCluster CR to set 'spec.devWorkspace.runningLimit' field"
```

Comment:
Does it revert fix of https://issues.redhat.com/browse/CRW-3012 ?

Reply:
No, script contained duplication of updating `runningLimits`. I've just remove it from a wrong place.